pages/conduct/faq.md

```diff
@@ -0,0 +1,46 @@
+<script>{
+	"title": "jQuery Foundation Code of Conduct - FAQ",
+	"pageTemplate": "page-conduct.php"
+}</script>
+
+This FAQ attempts to address common questions and concerns around the jQuery community's [Code of Conduct](https://www.jquery.org/conduct/). If you still have questions after reading it, please feel free to [contact us](mailto:conduct@jquery.com).
+
+### Why have you adopted a Code of Conduct?
+
+We think the jQuery community is awesome. If you're familiar with the jQuery community, you'll probably notice that the Code basically matches what we already do. Think of this as documentation: we're taking implicit expectations about behavior and making them explicit.
+
+We know that the jQuery community is open, friendly, and welcoming. We want to make sure everyone else knows it too.
+
+### What does it mean to "adopt" a Code of Conduct?
+
+For the most part, we don't think it means large changes. We think that the text does a really good job describing the way the jQuery community already conducts itself. We expect that most people will simple continue to behave in the awesome way they have for years.
+
+However, we do expect that people will abide by the spirit and words of the CoC when in "official" jQuery spaces. This code has been adopted by both the jQuery core team and by the jQuery Software Foundation. That means that it'll apply both in community spaces _and_ at jQuery Foundation events.
+
+In practice, this means the various jQuery IRC channels (<tt>#jquery</tt>, <tt>#jquery-dev</tt>, etc.), bug tracking and code review tools, and "official" jQuery events such as sprints. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+
+### What about events funded by the jQuery Software Foundation?
+
+This Code of Conduct also covers any events that the jQuery Foundation funds. However, events funded by the jQuery Foundation already [require a code of conduct](https://www.jquery.org/conduct/). Isn't this redundant?
```

Comment:
It seems like the second half of paragraph seems to be expanding upon the question above, rather than answering it, especially the "Isn't this redundant" part.  It's not until the second paragraph that the question really starts getting answer.  This is a break in the style established by the other FAQs, which seem to start answering the question immediately after the bolded text.  Maybe revise the initial question to be more along the lines of mentioning that jQuery events already require a code of conduct, and how is that different from this new Code of Conduct?

(Sorry if this is nitpicky, this is more the English Degree coming out)